elementary/clients/dbt/dbt_runner.py

```diff
@@ -47,10 +47,11 @@ def _run_command(
         if vars:
             json_vars = json.dumps(vars)
             dbt_command.extend(["--vars", json_vars])
+        log_msg = f"Running {' '.join(dbt_command)}"
         if not quiet:
```

Comment:
Not really related to the PR - but I'm not a fan of this "quiet" flag (I'd prefer that we'll change the logging level of the console handler based on a flag)

Reply:
hmm I see it's controlled per command and not by an outside flag, so maybe it's fine